core/src/main/java/org/apache/calcite/sql/validate/SqlValidator.java

```diff
@@ -912,15 +912,39 @@ interface Config {
      */
     Config withLenientOperatorLookup(boolean lenient);
 
-    /** Returns whether the validator allows measures to be used without the
-     * AGGREGATE function. Default is true. */
-    @Value.Default default boolean nakedMeasures() {
+    /** Returns whether the validator allows measures to be used without
+     * AGGREGATE function outside an aggregating query. Default is true.
+     */
+    @Value.Default default boolean nakedMeasuresOutsideAggregatingQuery() {
       return true;
     }
 
+    /** Sets whether the validator allows measures to be used without AGGREGATE
+     * function outside an aggregating query.
```

Comment:
I think 'in a non-aggregate query' would be clearer than 'outside an aggregating query' (assuming I understand you correctly).

Reply:
Also, we seem to use the term 'aggregate query' more than 'aggregating query' these days.

Reply:
I think I addressed this phrasing.